spec/design/skel/ARM/State_H.thy

```diff
@@ -28,13 +28,9 @@ where
 
 end
 
-context begin interpretation Arch .
-
-requalify_consts
+arch_requalify_consts (H)
   wordBits
```

Comment:
Same question, why can't `wordBits` be moved?

Reply:
If we want to move it, we have to move `#INCLUDE_HASKELL Data/WordLib.lhs all_bits NOT wordBits` as well... and then there's the question where to move it *to*... upwards, there's RegisterSet_H, but that's arch-specific as well. Downstream, we need this before Untyped_H (generic), Structures_H (generic), ArchVSpace_H (arch).
Since `Structures_H` (generic) pulls in `State_H` (arch), then that'd be the only place we could put the wordBits-using stuff along with the requalify. Does it make sense there?

Reply:
Hmm, my first reaction is that it makes more sense in `State_H` than in `Structures_H`, which doesn't help. It might be too hard to move now, but maybe a comment at least.

Reply:
Actually, due to how theories are laid out it won't even get to Structures_H, since ArchTypes_H already needs the definitions from that import. So I'm going with this comment:
```
(* Note: while this requalify and arch-generic Haskell import of WordLib.lhs could be moved to
   a generic theory, no good candidate theory exists at the moment. *)
```